src/lib/signature_lib/find_address/find_address.ml

```diff
@@ -0,0 +1,163 @@
+(** This module defines a tool that can be used to find invalid public keys
+    with some fixed prefix.
+
+    By invalid, we mean that the public key is not a valid point on the Mina
+    elliptic curve, and thus that the public key has no corresponding private
+    key.
+
+    The prefix to search for can be found as the first positional argument to
+    this command. If none is given, the default value used is
+    zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+
+    Usage: dune exec src/lib/find_address/find_address.exe [PREFIX_STRING].
+*)
+
+open Core_kernel
+open Signature_lib
+
+let debug = false
+
+let default_prefix = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+
+let desired_prefix =
+  try Sys.argv.(1)
```

Comment:
should there be a check that the command line argument contains only letters from `Base58_check.mina_alphabet`?

Reply:
I wanted to add this, but there didn't seem to be an easy way to get back the string from a `Base58.alphabet`, so I left it without.

Reply:
A hack would be to run `B58.decode` on each character, given that alphabet, see if it raises.